byzcoin/bcadmin/clicontracts/README.md

```diff
@@ -61,4 +61,32 @@ Invoke an addProof on a deferred contract:
 ```bash
 # The --hash and --instID values are given when we spawn the deferred contract
 bcadmin contract deferred invoke addProof --hash ... --instID ... --iid 0
+```
+
+**Working scenario**:
+
+```bash
+# Run the nodes, create roster and set up the config
+./run_nodes.sh -n 5 -c -t -v 2
+export BC_CONFIG=~/GitHub/cothority/conode
```

Comment:
If it is not specified, the key and the config files are saved in the current location. I like to specify it in order to have all the mess in one place.

Reply:
Hm, if `BC_CONFIG` is not set it will go in a specific configuration folder (e.g. with Linux it goes in `.config`). Usual people shouldn't have to worry about this. If it really goes to your current location, there's a bug to fix.

Reply:
I would also move this out of the README because people reading it might be confused and it might create two places where the configs are stored if they're not sure what they're doing.

Reply:
README updated! I also changed to an absolute path when running `./run_nodes.sh` since it might not be obvious where the script is.
@Gilthoniel yes you're right, when I checked I made the mistake to reset the BC_CONFIG with `export BC_CONFIG=""`, which is wrong.